### PR TITLE
Issue#1. Add static folder.

### PR DESCRIPTION
**Pull request by alvincases**
Static folder includes:
- gulp configuratio, css, sass, and node_modules.

### CODE REVIEW

**Comment by Luukyb** [comment]
Thanks for the PR. Few comments:
- The SASS files should be aggregated into one single CSS file. Adding too many CSS files into the HTML page would make it slow.
- Please exclude the following folders using a gitignore file: bower_components, node_modules and static/public/css/map/*.

File: static/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Static Drupalcamp 2018",
```
Comment (Luukyb) [inline static/package.json]: Could you update to "Drupalcamp Cebu 2018"?

File: static/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (Luukyb) [inline static/package.json]: Is this normal?

File: static/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Luukyb/drupalcampcebu2018.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (Luukyb) [inline static/package.json]: Our license is MIT

File: static/sass/style.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+@import "partials/base";
+@import "partials/content";
+@import "partials/footer";
+@import "partials/global";
```
Comment (Luukyb) [inline static/sass/style.scss]: Could you move global to be second, after base?

File: static/package.json (hunk 2 above)
Comment (alvincases) [reply static/package.json]: This is generated by default when runnning npm init.

Comment (alvincases) [reply static/package.json]: Done

File: static/package.json (hunk 3 above)
Comment (alvincases) [reply static/package.json]: Done

File: static/sass/style.scss (hunk 4 above)
Comment (alvincases) [reply static/sass/style.scss]: Done

File: static/package.json (hunk 1 above)
Comment (alvincases) [reply static/package.json]: Done

**Comment by Luukyb** [comment]
Thanks for the update @alvincases

**Pull request merged**